src/main/java/org/terasology/minion/move/MoveAlongPathNode.java

```diff
@@ -45,6 +46,9 @@
 
     @Override
     public void construct(Actor actor) {
+        // TODO: Temporary fix for injection malfunction in actions, ideally remove this in the future.
+        pathRenderSystem = CoreRegistry.get(PathRenderSystem.class);
```

Comment:
Is it worth adding this into a place for it to be reused? Rather than having the dependency fetch every time construct is called?